data/kahet/kahet missions.txt

```diff
@@ -303,6 +303,579 @@ mission "Ringworld Debris: Quarg"
+	npc accompany save
+		to spawn
+			has "Ka'het: Death in the Graveyard 2: offered"
+		system "Pantica"
+		government "Remnant"
+		personality timid launching escort
+		ship "Gull (Support)" "Late Season Dancing"
+
+mission "Ka'het: Death in the Graveyard 2"
+	invisible
+	landing
+	source "Aventine"
+	to offer
+		has "Ka'het: Death in the Graveyard 1: active"
+	on offer
+		conversation
+			`It must be winter here on <planet>: the chilly valleys that normally surround the starport are covered by a thick layer of snow, and you can see your ship's anti-icing fluids being sprayed automatically over the cockpit window. You find Plume waiting for you right on the landing pad he indicated to you on your way down, not far from a group of Pelicans and a lone Gull.`
+			`	"No better chance to skip the pleasantries, eh?" he begins, rubbing his arms in the cold. He seems severely under-dressed for the weather. You invite him to come aboard, but he stops at the airlock's door. "Sorry, but this time I already have my ride for the mission. We have prepared the Gull out there based on your instructions - the machinery we want to loot will need an appropriately large cargo hold."`
+			choice
+				`	"What happened with that Maeri'het?"`
+					goto what
+				`	"My fleet would have sufficed."`
+					to display
+						"total ships" > 1
+				`	"My ship would have sufficed."`
+					to display
+						"total ships" == 1
+			`	"Oh, I can believe that!" he replies, quickly glancing around the payload bay. "Simply, outfitting a ship for the mission would take time that we do not have right now.`
+			label what
+			`	"The Ka'het, as you know, are part mechanical and part biological. What may be news to you, however, is their biological self is suppressed most of the time: we have been able to determine that their stellar navigation and system management is completely left to their programming," he signs, with an unusual look of aversion in his eyes.`
+			`	"Still, like most biological beings, after their nominal lifetime of hundreds of years, they can start dying of old age. When this happens, they enter a hibernation state, calling for a Builder in the hope of being rescued. Of course, no help is coming.`
+			`	"We have observed," he continues while reading from a datapad, "other Ka'het try to patch them up as best as they can after some time, presumably following their own programming to do so. It is very unusual to find the corpse of one just laying there alone. We are interested in more or less anything that cannot be extracted from the carcass of one of their kind. Memory banks, logic terminals, and in general, all parts of their mechanical brain that usually will not survive critical battle damage."`
+			`	Despite the sign language, you notice that he seems strikingly halfhearted compared to the previous times you have encountered him. As he heads toward his Gull, the Late Season Dancing, you can't help but wonder why.`
```

Comment:
I think this might be telling the player what they're thinking a bit. The player will wonder anyway from the first sentence:
```suggestion
			`	Despite the sign language, you notice that he seems strikingly halfhearted compared to the previous times you have encountered him. He heads toward his Gull, the Late Season Dancing.`
```